src/main/java/blackjack/domain/game/BlackJackGame.java

```diff
@@ -0,0 +1,113 @@
+package blackjack.domain.game;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.DrawStrategy;
+import blackjack.domain.gamer.Bet;
+import blackjack.domain.gamer.Dealer;
+import blackjack.domain.gamer.Gamers;
+import blackjack.domain.gamer.Player;
+import blackjack.domain.result.BlackJackResult;
+import blackjack.dto.GameResultDto;
+import blackjack.dto.GamerDto;
+
+public class BlackJackGame {
+
+	public static final int INIT_DISTRIBUTION_COUNT = 2;
+	private static final int DEFAULT_EARNING = 0;
+	public static final int DEALER_OPEN_COUNT_FIRST = 1;
+
+	private final Gamers gamers;
+	private final DrawStrategy deck;
+
+	public BlackJackGame(List<String> names, BettingInjector betting, DrawStrategy deck) {
+		this.gamers = new Gamers(names, betting);
+		this.deck = deck;
+	}
+
+	public GameResultDto play(HitRequester hitOrStay, CardsChecker cardChecker) {
+		giveFirstCards();
+		checkFirstCardsOfGamers(cardChecker);
+		askPlayerHitOrStay(hitOrStay, cardChecker);
+		askDealerHitOrStay();
+		return createResult();
+	}
+
+	private void giveFirstCards() {
+		for (int i = 0; i < INIT_DISTRIBUTION_COUNT; i++) {
```

Comment:
인덱스 값을 활용하지 않는데 for문을 쓸 필요가 있나요?

```suggestion
	gamers.giveCardToAllGamers(this.deck);
	gamers.giveCardToAllGamers(this.deck);
```

Reply:
혹시 게임의 규칙이 처음에 나눠주는 카드가 2장이 아닌 다른 장수로 바꿀 때 변경을 좀 더 용이하게 하기 위해서 이렇게 해주었습니다. 그럼에도 index를 활용해야지만 for문을 사용해야할까요?